data/basics/userBalance.go

```diff
@@ -56,7 +56,7 @@ const (
 	// EncodedMaxAppLocalStates is the decoder limit for number of opted-in apps in a single account.
 	// It is verified in TestEncodedAccountAllocationBounds to align with
 	// config.Consensus[protocol.ConsensusCurrentVersion].MaxppsOptedIn
-	EncodedMaxAppLocalStates = 64
+	EncodedMaxAppLocalStates = 6000001
```

Comment:
`		ad, err := client.AccountData(nacc.pk.String()) `
will return an error if this bound is not increased.
There are similar parameters here which are not tested for unlimited apps/accounts.